src/util/apiUtils.js

```diff
@@ -227,6 +223,9 @@ export function parseRequest(request, baseUrl) {
 	delete externalRequestHeaders['host'];
 	delete externalRequestHeaders['accept-encoding'];
 	delete externalRequestHeaders['content-length'];  // original request content-length is irrelevant
+	delete externalRequestHeaders['cf-ray']; // unique cloudflare id which will trip up api requests if present
+	
+	console.log(`External request headers: ${JSON.stringify(externalRequestHeaders)}`);
```

Comment:
sorry to confuse - `externalRequestHeaders` is set up here, but `buildRequestArgs` adds a few more, so this log won't catch `'X-Meta-Photo-Host'` and `'X-Meetup-Request-Flags'`, for example - that won't affect what you're inspecting, though, so i'll merge